Gems/MotionMatching/Code/Tests/MinMaxScalerTests.cpp

```diff
@@ -74,6 +74,63 @@ namespace EMotionFX::MotionMatching
         EXPECT_EQ(t(2,0), 1.0f); EXPECT_EQ(t(2, 1), 0.75f);EXPECT_EQ(t(2, 2), 0.25f);EXPECT_EQ(t(1, 3), 3.0f);
     }
 
+    TEST_F(MinMaxScalerFixture, TransformValueNoClipping)
+    {
+        FeatureMatrix m;
+        m.resize(2, 1);
+        m(0, 0) = -2.0f;
+        m(1, 0) = 2.0f; // range = 4.0
+
+        MinMaxScaler minMaxScaler;
+        EXPECT_TRUE(minMaxScaler.Fit(m));
+
+        EXPECT_EQ(minMaxScaler.Transform(-6.0f, 0), -1.0f);
+        EXPECT_EQ(minMaxScaler.Transform(4.0f, 0), 1.5f);
+    }
+
+    TEST_F(MinMaxScalerFixture, TransformValueClip)
+    {
+        FeatureMatrix m;
+        m.resize(2, 1);
+        m(0, 0) = -2.0f;
+        m(1, 0) = 2.0f;
+
+        MinMaxScaler minMaxScaler;
+        EXPECT_TRUE(minMaxScaler.Fit(m, {/*featureMin=*/0.0f, /*featureMax=*/1.0f, /*clip=*/true}));
```

Comment:
nit: Provide the object name with the initializer list here - when greping code to find occurrences it does make things so much easier (I've used this style in the past and lived to regret it 😅)